src/inverseRobot.js

```diff
@@ -25,6 +25,19 @@
  */
 function inverseRobot(robot) {
   // write code here
+  const inverse = {};
```

Comment:
I think this one looks better
```suggestion
  const inversedRobot = {};
```